service/src/registry.ts

```diff
@@ -399,7 +417,7 @@ export class Registry {
    * @param {Function}    done    The function to call with two arguments: an error object if an error happened,
    *                              (null otherwise) and the list of past jobs as an argument.
    */
-  listJobs(done: Callback<any>): void {
+  listJobs(done: IncomingMessageCallback<any>): void {
```

Comment:
are we sure these callback type changes are non-breaking even for typescript users?

Reply:
Yes, the generated JavaScript will be identical and TS is smart enough to interfere the new type.

Reply:
Done

---
In reply to: [219359411](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r219359411) [](ancestors = 219359411)